dapps/marketplace/src/pages/identity/EmailAttestation.js

```diff
@@ -180,11 +180,14 @@ class EmailAttestation extends Component {
             <div className="my-3 verification-code">
               <input
                 type="tel"
+                maxLength="6"
                 ref={ref => (this.inputRef = ref)}
                 className="form-control form-control-lg"
                 placeholder="Verification code"
                 value={this.state.code}
-                onChange={e => this.setState({ code: e.target.value })}
+                onChange={e =>
```

Comment:
Can we show an error if the user tries to submit a non-numeric code instead of just disallowing non-numeric characters? User may wonder why they can't type otherwise

Reply:
Sure. Will make that change

Reply:
Done.